files/build_templates/per_namespace/lldp.service.j2

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=LLDP container
+Requires=database{% if multi_instance == 'true' %}@%i{% endif %}.service
+Requires=updategraph.service
+After=database{% if multi_instance == 'true' %}@%i{% endif %}.service
+After=swss{% if multi_instance == 'true' %}@%i{% endif %}.service
+After=syncd{% if multi_instance == 'true' %}@%i{% endif %}.service
+After=updategraph.service
```

Comment:
Suggest to move this line under line 4 to keep it together with the `Requires=updategraph.service` line.

Reply:
updated as per comment.